gulpfile.js

```diff
@@ -151,7 +190,7 @@ function buildUid() {
   return gulp.src(['src/ssp-userids/uid.js'])
   .pipe(webpackStream(cloned))
     .pipe(uglify())
-    .pipe(header('/* v<%= creative.version %>\n' + dateString + ' */\n', { creative: creative }))
+    .pipe(header('/* v<%= creative.version %>\n' + dateString + '\nDEPRECATED, please use creative based on hb_format targeting */\n', { creative: creative }))
```

Comment:
Is this comment in the right prod build function?  I might have missed it, but what hb_format does the uid.js file connect with?  Was this supposed to go in one of the older native prod files?